registration/templates/registration/login.html

```diff
@@ -7,6 +7,7 @@
 <form method="post" action="">
     {% csrf_token %}
     {{ form.as_p }}
+    <p><label><input type="checkbox" name="remember_me" />{% trans "Remember Me" %}</label></p>
```

Comment:
if we include the remember_me in the form, we don't need to manually add the line here